.env

```diff
@@ -1,5 +1,5 @@
-VUE_APP_TITLE=National Integrated Water Availability Assessments
+VUE_APP_TITLE=National Integrated Water Availability Assessments: Water Availability Mapper
 
-VUE_APP_TITLE_SUFFIX=-Concept Map
+VUE_APP_TITLE_SUFFIX= 
```

Comment:
wondering if this suffix is baked into any other places in the app, if so, we might want to just remove it?

Reply:
As far as I can tell it was only for the title?  So I can see removing it being fine.  Do you see anything weird without it?

Reply:
Removing it locally had no noticeable issues and console is clean.  Meh, can always puyt it back if we find soemthing later